src/components/SearchPane/_search-pane.scss

```diff
@@ -0,0 +1,210 @@
+// ----------------------------------
+// UI Components
+// Search Pane
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2020-01-20
+// ----------------------------------
+
+// Variables
+
+$transition-duration: 0.4s;
+
+// Extends
+
```

Comment:
Looks like we have some duplication with `%nav-btn-base` in `_header.scss`

Doesn't need addressing in this PR but we should discuss amalgamating the two